SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -40,16 +40,39 @@
     </style>
   </head>
   <body>
-  <h1>{{ currentQuestion.title }}</h1>
-  <p>{{ currentQuestion.content }}</p>
-  <h2>Question by {{ currentQuestion.owner }}</h2>
-    <div id="wmd-button-bar"></div>
-    <textarea id="wmd-input" class="wmd-input"></textarea>
-    <div id="wmd-preview" class="wmd-panel wmd-preview"></div>
-  <script>
-      var converter = Markdown.getSanitizingConverter();
-      var editor = new Markdown.Editor(converter);
-      editor.run();
-    </script>
+  <!-- Display the question's data -->
+  <div id="question">
+      <h1>{{ currentQuestion.title }}</h1>
+      <p>{{ currentQuestion.content }}</p>
+      <h2>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date }}</h2>
+  </div>
+  <div id="answers">
+  <!-- Display the answers -->
```

Comment:
Great idea to show metadata for the question as well, something we all forgot to do.